backend/app_sdk/app_base.py

```diff
@@ -288,6 +288,16 @@ def split(base, sep):
         return base.split(sep)
 
 
+@shuffle_filters.register
+def url_encode(base):
```

Comment:
How does this solve the problem? Isn't this 100% the same as the implementation they already had? Doesn't this lead to + instead of %20?